charts/consul/templates/connect-inject-deployment.yaml

```diff
@@ -364,6 +364,18 @@ spec:
               {{- if .Values.global.adminPartitions.enabled }}
               -partition={{ .Values.global.adminPartitions.name }} \
               {{- end }}
+              {{- if .Values.externalServers.enabled }}
+              {{- if .Values.global.tls.enabled }}
```

Comment:
[non-blocking] would would you think about putting this in a template since it seems to be repeated quite a bit?

Reply:
yeah great question! I think I'll actually be able to get rid of this init container from all components altogether once agentless is done. My plan is to move login/logout logic into controllers to do at startup and shutdown. Other components that currently use this will switch to consul-dataplane instead doing the login for them.